build.sbt

```diff
@@ -23,7 +23,7 @@ val scalacVersion = "2.13.11"
 val graalVersion  = "17.0.7"
 // Version used for the Graal/Truffle related Maven packages
 val graalMavenPackagesVersion = "23.0.0"
-val targetJavaVersion         = graalVersion.split("\\.")(0)
+val targetJavaVersion         = "11"
```

Comment:
Better to target JDK-11 - the [Bck2Brwsr Transpiler](https://github.com/jtulach/bck2brwsr/) hasn't really been testing with something newer.